v1.0/conformance_test_v1.0.yaml

```diff
@@ -981,3 +981,25 @@
       size: 15
   tool: v1.0/imported-hint.cwl
   doc: Test hints with $import
+
+- job: v1.0/recursive-input-directory.json
+  output:
+    "outdir": {
```

Comment:
you mean the `"outdir": {` is not yaml syntax? The same syntax is used in the same file?